src/types/Plan.ts

```diff
@@ -21,13 +21,28 @@ export interface Plan extends PlanReference {
     total_before_discount: number;
     total_after_discount: number;
     usage: Usage[];
+    possible_changes: Change[];
+    ends_at: string | null;
+    ended_at: string | null;
 }
 
 export interface Usage {
     limit: Limit;
     used: number;
 }
 
+export interface Change {
+    pricing_table_option_id: string;
```

Comment:
What should i do to get list of plans card?

Reply:
It's in the Pricing Tables API. You just list the tables from there, you will get all the pricing options.